input/fsh/CodeSystemAliases.fsh

```diff
@@ -23,3 +23,7 @@ Alias:   ObsCat = http://terminology.hl7.org/CodeSystem/observation-category
 Alias:   ObsInt = http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation
 Alias:   DiagnosticService = http://terminology.hl7.org/CodeSystem/v2-0074
 Alias:   OmbRaceCat = urn:oid:2.16.840.1.113883.6.238
+Alias:   UnitsOfTime = http://hl7.org/fhir/ValueSet/units-of-time
+Alias:   TimingAbbreviation = http://terminology.hl7.org/CodeSystem/v3-GTSAbbreviation
+Alias:   MedRequestStatus = http://hl7.org/fhir/ValueSet/medicationrequest-status
```

Comment:
Change to the code system http://hl7.org/fhir/CodeSystem/medicationrequest-status

Reply:
yeh...those are bugs. I'll fix.